6 mls wheat

SinaZumst opened the pull request:
Can this be merged like this already? If there are changes to implement please let me know.

lwiniwar reviewed (commented):
for the example 5 thumbnail, please change "TLS" to "ALS" :)

lwiniwar commented on README.md:
```diff
@@ -60,9 +60,12 @@ keywords = {Software, LiDAR simulation, Point cloud, Data generation, Voxel, Veg
+| [![Example 1](img/example1_thumbnail.png)](https://nbviewer.org/github/3dgeo-heidelberg/helios/blob/dev/example_notebooks/1-getting-started.ipynb)        | [![Example 2](img/example2_thumbnail.png)](https://nbviewer.org/github/3dgeo-heidelberg/helios/blob/dev/example_notebooks/2-the-survey.ipynb)            |
+|-----------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [![Example 3](img/example3_thumbnail.png)](https://nbviewer.org/github/3dgeo-heidelberg/helios/blob/dev/example_notebooks/3_arbaro_TLS_demo.ipynb)        | [![Example A](img/exampleA_thumbnail.png)](https://nbviewer.org/github/3dgeo-heidelberg/helios/blob/dev/example_notebooks/arboretum_notebook.ipynb)      |
+| --------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
```
The spaces in line 66 (after the | and before the ---, and in the middle) break the table formatting on github.

han16nah commented on example_notebooks/5_ALS_hd_demo.ipynb:
```diff
@@ -0,0 +1,1657 @@
+{
```
<p>Did you create the <code>6-mls-wheat</code> branch from the previous feature branch (instead of from <code>dev</code>) or why is this in here?    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

SinaZumst replied:
That might have been a mistake, is this a big problem?

SinaZumst replied:
would it be the best thing to delete the files from the 6_mls_wheat branch or would that cause problems when merging the branch to dev?

han16nah replied:
No, it's not a problem. When merging, we will see that there is a merge conflict and then we simply choose whether to keep the file from the dev branch or from your feature branch :)

han16nah commented on example_notebooks/6_mls_wheat_demo.ipynb:
```diff
@@ -0,0 +1,2666 @@
+{
```
<p>Here, we should add something about the turn types of the platform, because they are interesting:</p><p>The ground vehicle tries to mimic real vehicles. It performs "smooth turns" for wide-angle curves. For narrow-angle curves, it first turns, then backs up, then finishes the turn (2 point turn). This can also be seen in the trajectory and we might also just give a short teaser here and add the detailed explanation below the figure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

han16nah replied:
<p>I don't think further details are necessary, because there is nothing too specific for this survey.. We could add links to the Wiki throughout the notebook(s) though :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

han16nah replied:
<p>Here, we should add that the same object (the wheat plant) is placed in the scene 15 times. Using the translate filter, it is positioned in three columns and five rows to form a wheat field.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

SinaZumst commented:
I included the suggested changes, thank you. I deleted the old files that didn't belong in this branch and corrected the thumbnail for the 5_hd_demo in a seperate branch. I hope this works now
